assets/queries/dockerfile/y_switch_avoid_manual_input/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "y_switch_avoid_manual_input",
+  "queryName": "Y Switch Avoid Manual Input",
```

Comment:
The query name should reflect the vulnerability in an easy and comprehensible way
"yum_install_allows_manual_input"